### ISSUE TITLE
FR: please co-publish on Open VSX Registry

### ISSUE THREAD

**Issue by GitMensch** [jsonreeder/vscode-org-mode#175]
The [Open VSX Registry](https://open-vsx.org/?search=org) currently does not include any extension for org files. It is the default extension provider endpoint for the free build of vscode VSCodium and Eclipse Theia.
It would be nice to have this extension be directly part of a free vsix repository instead of the Microsoft Marketplace with its restrictions.

Details on the publishing can be found at https://github.com/eclipse/openvsx/wiki/Publishing-Extensions.

**Comment by KlavsKlavsen** [comment]
Really would be nice to have :)

**Comment by Gnurou** [comment]
Yup, that's actually what is preventing from using this extension at the moment.

**Comment by GitMensch** [comment]
@KlavsKlavsen actually used the reasonable way to add extensions if the original author does not do this - the CI build. Normally the extensions should be available now but something prevents that, see https://github.com/open-vsx/publish-extensions/pull/334.

Ideally there would be a 1.1 release of this extension with a vsix included in the release page and/or the person that would upload the 1.1 to the ms-market place would also do this to the Open VSX Registry.

**Comment by mrkeuz** [comment]
+1 Currently cannot easily install into `gitpod/openvscode-server`. Just have personal org wiki and want to edit it from browser.

As WORKAROUND for vscode-server:
1. Download from `*.vsix` from marketplace
    https://marketplace.visualstudio.com/items?itemName=vscode-org-mode.org-mode
2. Click `Install from VSIX...`

See: https://www.gitpod.io/docs/vscode-extensions/#where-do-i-find-extensions

**Comment by anddam** [comment]
Download of vsix is not available anymore on Visual Studio Marketplace.
This extension is nowhere to be found on either the [Visual Studio Marketplace](https://marketplace.visualstudio.com/search?term=org-mode&target=VSCode&category=All%20categories&sortBy=Relevance) nor on the [Open VSX Registry](https://open-vsx.org/?search=org-mode&sortBy=relevance&sortOrder=desc).

The `tootone.vscode-org-mode` is available on VS Marketplace but I found it to behave differently, for instance it does not fold outlines (very uncomfortable).

What is needed to make this available again, especially on Open VSX Registry?
